Source/Charts/Animation/ChartAnimationEasing.swift

```diff
@@ -341,7 +341,7 @@ internal struct EasingFunctions
         let s: TimeInterval = 1.70158
         var position: TimeInterval = elapsed / duration
         position -= 1.0
-        return Double( position * position * ((s + 1.0) * position + s) + 1.0 )
+        return Double( position * position * ((s + Double(1.0)) * position + s) + Double(1.0) )
```

Comment:
Would be better if `s` and `position` typecasted before calculate ?
Or 1.0 would be typecasted once ?)

Reply:
Updated, just remove Double cast and Xcode can build the project @iDevPro